### ISSUE TITLE
Adopting Loss truncation for ASR 

### ISSUE THREAD

**Issue by Amg9794** [ddkang/loss_dropper#6]
Hi 

can this  be adopted for ASR (trained with CE loss) task ?

**Comment by ddkang** [comment]
Our technique is fairly generic so likely will apply to any setting that uses cross-entropy. However, the specific details may differ depending on exactly how the loss is computed.
